app/tests/e2e/test_connections.py

```diff
@@ -109,6 +109,55 @@ async def test_get_connections(
     assert_that(len(alice_connections)).is_greater_than_or_equal_to(1)
     assert_that(len(bob_connections)).is_greater_than_or_equal_to(1)
 
+    alice_alias = (await alice_member_client.get(f"{BASE_PATH}?alias=alice")).json()[0][
+        "alias"
+    ]
+    assert alice_alias == "alice"
+
+    alice_state = (
+        await alice_member_client.get(f"{BASE_PATH}?state=completed")
+    ).json()[0]["state"]
+    assert alice_state == "completed"
+
+    alice_key = (
+        await alice_member_client.get(
+            f"{BASE_PATH}?invitation_key={bob_connections[0]['invitation_key']}"
+        )
+    ).json()[0]["invitation_key"]
+    assert alice_key == alice_connections[0]["invitation_key"]
+
+    alice_invitation_msg_id = (
+        await alice_member_client.get(
+            f"{BASE_PATH}?invitation_msg_id={alice_connections[0]['invitation_msg_id']}"
+        )
+    ).json()[0]["invitation_msg_id"]
+    assert alice_invitation_msg_id == alice_connections[0]["invitation_msg_id"]
+
+    alice_my_did = (
+        await alice_member_client.get(
+            f"{BASE_PATH}?my_did={alice_connections[0]['my_did']}"
+        )
+    ).json()[0]["my_did"]
+    assert alice_my_did == alice_connections[0]["my_did"]
+
+    alice_their_did = (
+        await alice_member_client.get(
+            f"{BASE_PATH}?their_did={bob_connections[0]['my_did']}"
+        )
+    ).json()[0]["their_did"]
+    assert alice_their_did == alice_connections[0]["their_did"]
+
+    with pytest.raises(HTTPException) as exc:
+        await alice_member_client.get(
+            f"{BASE_PATH}?their_public_did={bob_connections[0]['their_public_did']}"
+        )
+    assert exc.value.status_code == 422
+
+    alice_their_role = (
+        await alice_member_client.get(f"{BASE_PATH}?their_role=inviter")
+    ).json()[0]["their_role"]
+    assert alice_their_role == "inviter"
+
```

Comment:
Just apply a lil bit of deduplication here: namely with all the `bob_connections[0]` and `alice_connections[0]` calls. The [0] can be dropped just by re-assigning, e.g., `alice_connection = alice_connections[0]`. And any other fields that are duplicated, like `alice_connections[0]["my_did"]` and `["invitation_msg_id"]`